### DISCUSSION TITLE
trRosettaRNA weights from server used in source code

### DISCUSSION THREAD

**Discussion by ReaKal** [RosettaCommons/rosetta#194] [ROSIE]
Can the weights of each of the 5 models downloaded from
https://yanglab.qd.sdu.edu.cn/trRosettaRNA/benchmark/
be used locally with the source code of trRosettaRNA v1.1 provided at https://zenodo.org/records/10616895?

Based on the calculation made below, the state_dict of the model_1.pth.tar trained to the 201012 dataset, which was downloaded from https://yanglab.qd.sdu.edu.cn/trRosettaRNA/benchmark/, includes 6556 keys, while  the state_dict of the trRosettaRNA_v1.1/params/model_1/models/model_1.pth.tar, which was downloaded from [zenodo](https://zenodo.org/records/10616895), has 2612 keys.

```
import torch

tar_file_1 = /path/to/trRosettaRNA_v1.1/params/model_1/models/model_1.pth.tar
tar_file_2 = /path/to/model_201012/state_dict/model_1.pth.tar

state_dict_1 = torch.load(tar_file_1)
state_dict_2 = torch.load(tar_file_2)['state_dict']

print(len(list(state_dict_1.keys())))
print(len(list(state_dict_2.keys())))
```

Thank you,
Rea

**Comment by lyskov** [comment]
Out of scope and non-actionable at this time — closing for now.
